MODULE_ACCEPTANCE_CRITERIA.MD

```diff
@@ -1,9 +1,10 @@
 # FOLIO Module Acceptance Values and Criteria
 
-## version 1.1 (ratified TBD)
+## version 2.0 (ratified TBD)
```

Comment:
> interesting idea. Maybe we can use tags for this, or explicitly make copies for archival purposes. 

I'd be happy with something like tags.

It was an open question. I hadn't really thought forward about the how or why.

> When do you see older versions being referenced?

I might think that folks could be interested in what criteria applied for a release or for a given approval (I know these are copied to the review itself, so that might be sufficient).

Reply:
holding off on this for now.  Maybe something to look at next iteration of this